<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumItemDialog</class>
 <widget class="QDialog" name="NumItemDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>436</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../penyamanager.qrc">
    <normaloff>:/images/alegrialogo.jpg</normaloff>:/images/alegrialogo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#NumItemDialog {
background: gray;
}

#numDisplayWidget {
border: 3px solid gray;
border-radius: 20px;
background: white;

}

#numDisplayLabel {
font-weight: bold;
font-size: 15px;
}

QToolButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 15px;
padding-left: 5px;
padding-right: 5px;
min-width: 95px;
max-width: 95px;
min-height: 95px;
max-height: 95px;
}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="numDisplayWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="numDisplayLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="keypadWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="3">
       <widget class="QToolButton" name="toolButton_8">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_1">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButton_2">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="toolButton_3">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="toolButton_4">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="toolButton_5">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QToolButton" name="toolButton_6">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QToolButton" name="toolButton_9">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="toolButton_7">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QToolButton" name="toolButton_C">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QToolButton" name="toolButton_Done">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QToolButton" name="toolButton_0">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>103</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../penyamanager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_Done</sender>
   <signal>clicked()</signal>
   <receiver>NumItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
